crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -115,8 +116,6 @@ impl Plugin for MeshRenderPlugin {
         load_internal_asset!(app, SKINNING_HANDLE, "skinning.wgsl", Shader::from_wgsl);
         load_internal_asset!(app, MORPH_HANDLE, "morph.wgsl", Shader::from_wgsl);
 
-        app.add_plugins(GpuComponentArrayBufferPlugin::<MeshUniform>::default());
```

Comment:
Why was this removed?

Reply:
Building the MeshUniform buffer is now done in `prepare_mesh_uniform_buffer` so it can be done in sorted order.